stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -215,13 +215,15 @@ public synchronized void addAdditionalResult() throws IOException {
         return;
       }
       Map<String, MethodStatistics> nameStatistics =
-          processMethodProfiles(mResult.getRecordStartMs(), mResult.getEndMs(), (type, method) -> {
-            if (type.contains("RPC")) {
-              final int classNameDivider = method.lastIndexOf(".");
-              method = method.substring(classNameDivider + 1);
-            }
-            return type + ":" + method;
-          });
+          processMethodProfiles(mResult.getRecordStartMs(), mResult.getEndMs(),
+              javaAgentInput -> {
```

Comment:
If the class was renamed, then this would probably be `profileInput`

Reply:
changed.